pkg/kubelet/metrics/collectors/volume_stats.go

```diff
@@ -120,6 +120,12 @@ func (collector *volumeStatsCollector) CollectWithStability(ch chan<- metrics.Me
 			addGauge(volumeStatsInodesDesc, pvcRef, float64(*volumeStat.Inodes))
 			addGauge(volumeStatsInodesFreeDesc, pvcRef, float64(*volumeStat.InodesFree))
 			addGauge(volumeStatsInodesUsedDesc, pvcRef, float64(*volumeStat.InodesUsed))
+<<<<<<< Updated upstream
+=======
+			if volumeStat.VolumeHealthStats != nil {
+				addGauge(volumeStatsHealthAbnormalDesc, pvcRef, convertBoolToFloat64(volumeStat.VolumeHealthStats.Abnormal))
+			}
+>>>>>>> Stashed changes
```

Comment:
Can you squash the "[resolve conflict](https://github.com/kubernetes/kubernetes/pull/108758/commits/afdbd27abfef8a10cc4ba389dbc2fd68ba70535f)" commit into this one?

Reply:
sure